adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/WebUI.cs

```diff
@@ -48,11 +48,16 @@ public async Task<AuthorizationResult> AcquireAuthorizationAsync(Uri authorizati
         {
             AuthorizationResult authorizationResult = null;
 
-            var sendAuthorizeRequest = new Action(
-                delegate
-                {
-                    authorizationResult = this.Authenticate(authorizationUri, redirectUri);
-                });
+            var sendAuthorizeRequest = new Action(() =>
+            {
+                var other = PlatformParameters._syncContext;
```

Comment:
>_syncContext [](start = 47, length = 12)

alternatively it should be done by invoking the call on the ownerwindow thread?

Reply:
@henrik-me - we expect the ownerwindow to be a Handle (intptr handle), see https://github.com:443/AzureAD/azure-activedirectory-library-for-dotnet.git/blob/master/adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs#L94

Also, it seems the "ownerwindow" is only used to center the dialog, and in most our samples it is null. So we can expect most people will not have set it.